.changeset/lemon-gifts-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': minor
```

Comment:
should be major since this is breaking: https://github.com/interledger/open-payments/pull/461/files#diff-2d8be9fd3b70cc62b8a63016a7bf60f5d21f191b9e238966a0ed7170957a77d6R36